dusk.md

```diff
@@ -1533,9 +1533,9 @@ As shown above, a "date picker" is an example of a component that might exist th
         {
             return [
                 '@date-field' => 'input.datepicker-input',
-                '@year-list' => 'div > div.datepicker-years',
+                '@year-list'  => 'div > div.datepicker-years',
                 '@month-list' => 'div > div.datepicker-months',
-                '@day-list' => 'div > div.datepicker-days',
+                '@day-list'   => 'div > div.datepicker-days',
```

Comment:
Please revert all indenting here.

Reply:
Apologies - I thought your standard was to align. Fixed now

Reply:
Did you see this anywhere else? Otherwise we can update those places as well (in a different pr/commit).

Reply:
Checked through all the docs and instances occur in [`database.md`](https://github.com/laravel/docs/blob/7.x/database.md), [`passport.md`](https://github.com/laravel/docs/blob/7.x/passport.md) and [`validation.md`](https://github.com/laravel/docs/blob/7.x/validation.md).

Also, on my travels I found instances where trailing commas were missing from arrays and 10 files with trailing whitespace that hasn't been trimmed and may cause a conflict in a future merge.

PRs ready, just let me know which corrections you'll accept!

Reply:
@u01jmg3 feel free to send in a different pr for those.

Reply:
Done